src/test/kotlin/three/consulting/epoc/repository/ProjectServiceIntegrationTest.kt

```diff
@@ -113,4 +113,9 @@ class ProjectServiceIntegrationTest : IntegrationTest() {
             .isInstanceOf(UnableToDeleteProjectException::class.java)
             .hasMessage("Cannot delete project, no project found for given id: 1000")
     }
+
+    @Test
+    fun `get all projects`() {
+        assertThat(projectService.findAllProjects()).isNotNull
```

Comment:
Please assert in more detail than this. Probably you're expecting to find something and checking `isNotNull` doesn't verify if it's a unit or an empty list. Maybe checking that the names of the returned projects are something you're expecting? So something like
```kotlin
val projects = projectService.findAllProjects()
assertThat(projects.map { it.name }).containsOnlyElementsOf(listOf("This projetct", "That project"))
```
or something like that